### DISCUSSION TITLE
[Action Required] Upgrade Applications using AWS SDK for Java to Version 1.12.x or Later

### DISCUSSION THREAD

**Discussion by gvoden** [aws/aws-sdk-java#2823] [Q&A] [answered]
Hi,

we got notified by AWS that we are using older Java SDK versions and need to update in order to maintain support for TLS 1.3. 
Is there a current road map of which AWS services will start enforcing TLS 1.3 vs 1.2. This would help us assess the urgency of this SDK update.

thanks!

**Comment by yasminetalby** [comment, marked as answer]
Hello @gvoden,

Thank you very much for your message.
Older versions of the [AWS SDK for Java,](https://github.com/aws/aws-sdk-java) 1.9.5 to 1.10.31, are incompatible with TLS 1.3.
The update has been sent because on July 19th, 2022 Java 8 [enabled TLS 1.3 by default](https://java.com/en/jre-jdk-cryptoroadmap.html). As of May 11, 2022, versions of the JDK from 11 onwards enable TLS 1.3 by default.

This [article](https://aws.amazon.com/blogs/developer/tls-1-3-incompatibility-with-aws-sdk-for-java-versions-1-9-5-to-1-10-31/) covers all the detail concerning the TLS 1.3 incompatibility such as:
- how to check if your application is impacted
- steps to take if your application is impacted

To note that this will not impact you if you are not using versions of the JDK from 8 onwards.

Let me know if you have any other questions.

Best,

Yasmine
